2025-06-13 02:02:37.955Z |  INFO     | storeapi.main:20 - Starting application lifespan
2025-06-13 02:02:37.958Z |  DEBUG    | storeapi.main:21 - Connecting to the database
2025-06-13 02:02:38.223Z |  INFO     | uvicorn.error:62 - Application startup complete.
2025-06-13 02:02:38.224Z |  INFO     | uvicorn.error:215 - Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
2025-06-13 02:02:55.725Z |  INFO     | uvicorn.access:476 - 127.0.0.1:3999 - "GET /posts/ HTTP/1.1" 200
2025-06-13 02:03:46.346Z |  INFO     | uvicorn.access:476 - 127.0.0.1:4023 - "POST /posts/ HTTP/1.1" 201
2025-06-13 02:04:52.780Z |  INFO     | uvicorn.error:263 - Shutting down
2025-06-13 02:04:52.882Z |  INFO     | uvicorn.error:67 - Waiting for application shutdown.
2025-06-13 02:04:52.886Z |  INFO     | uvicorn.error:76 - Application shutdown complete.
2025-06-13 02:04:52.888Z |  INFO     | uvicorn.error:93 - Finished server process [3140]
2025-06-14 11:43:49.428Z |  INFO     | storeapi.main:21 - Starting application lifespan
2025-06-14 11:43:49.431Z |  DEBUG    | storeapi.main:22 - Connecting to the database
2025-06-14 11:43:49.704Z |  INFO     | uvicorn.error:62 - Application startup complete.
2025-06-14 11:43:49.704Z |  INFO     | uvicorn.error:215 - Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
2025-06-14 11:43:57.048Z |  INFO     | uvicorn.access:476 - 127.0.0.1:1514 - "GET /posts/1/ HTTP/1.1" 307
2025-06-14 11:43:57.059Z |  INFO     | uvicorn.access:476 - 127.0.0.1:1514 - "GET /posts/1 HTTP/1.1" 200
2025-06-14 11:44:01.135Z |  INFO     | uvicorn.access:476 - 127.0.0.1:1514 - "GET /posts/10/ HTTP/1.1" 307
2025-06-14 11:44:01.140Z |  ERROR    | storeapi.main:55 - Status: 404 HTTP Exception: Post not found
2025-06-14 11:44:01.142Z |  INFO     | uvicorn.access:476 - 127.0.0.1:1514 - "GET /posts/10 HTTP/1.1" 404
2025-06-14 11:58:04.936Z |  INFO     | uvicorn.error:263 - Shutting down
2025-06-14 11:58:05.039Z |  INFO     | uvicorn.error:67 - Waiting for application shutdown.
2025-06-14 11:58:05.041Z |  INFO     | uvicorn.error:76 - Application shutdown complete.
2025-06-14 11:58:05.042Z |  INFO     | uvicorn.error:93 - Finished server process [26508]
2025-06-14 12:28:27.477Z | (-) | INFO     | storeapi.main:22 - Starting application lifespan
2025-06-14 12:28:27.481Z | (-) | DEBUG    | storeapi.main:23 - Connecting to the database
2025-06-14 12:28:27.745Z | (-) | INFO     | uvicorn.error:62 - Application startup complete.
2025-06-14 12:28:27.747Z | (-) | INFO     | uvicorn.error:215 - Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
2025-06-14 12:28:35.632Z | (54752d05) | INFO     | uvicorn.access:476 - 127.0.0.1:2757 - "GET /posts/10/ HTTP/1.1" 307
2025-06-14 12:28:35.638Z | (19804884) | ERROR    | storeapi.main:58 - Status: 404 HTTP Exception: Post not found
2025-06-14 12:28:35.639Z | (19804884) | INFO     | uvicorn.access:476 - 127.0.0.1:2757 - "GET /posts/10 HTTP/1.1" 404
2025-06-14 12:29:04.467Z | (2e17cd83) | INFO     | uvicorn.access:476 - 127.0.0.1:2780 - "GET /posts/10/ HTTP/1.1" 307
2025-06-14 12:29:04.472Z | (4a6a2dec) | ERROR    | storeapi.main:58 - Status: 404 HTTP Exception: Post not found
2025-06-14 12:29:04.474Z | (4a6a2dec) | INFO     | uvicorn.access:476 - 127.0.0.1:2780 - "GET /posts/10 HTTP/1.1" 404
2025-06-14 12:29:04.524Z | (b84e6659) | INFO     | uvicorn.access:476 - 127.0.0.1:2780 - "GET /favicon.ico HTTP/1.1" 404
2025-06-14 12:29:41.648Z | (e0e49f75) | INFO     | uvicorn.access:476 - 127.0.0.1:2790 - "GET /posts/10/ HTTP/1.1" 307
2025-06-14 12:29:41.650Z | (824c565e) | ERROR    | storeapi.main:58 - Status: 404 HTTP Exception: Post not found
2025-06-14 12:29:41.652Z | (824c565e) | INFO     | uvicorn.access:476 - 127.0.0.1:2790 - "GET /posts/10 HTTP/1.1" 404
2025-06-14 12:29:55.704Z | (3f17d1ad) | INFO     | uvicorn.access:476 - 127.0.0.1:2797 - "GET /posts/10/ HTTP/1.1" 307
2025-06-14 12:29:55.707Z | (1d64c01d) | ERROR    | storeapi.main:58 - Status: 404 HTTP Exception: Post not found
2025-06-14 12:29:55.709Z | (1d64c01d) | INFO     | uvicorn.access:476 - 127.0.0.1:2797 - "GET /posts/10 HTTP/1.1" 404
2025-06-14 12:31:20.033Z | (24cb4021) | INFO     | uvicorn.access:476 - 127.0.0.1:2995 - "GET /posts/10/ HTTP/1.1" 307
2025-06-14 12:31:20.036Z | (527bf3d6) | ERROR    | storeapi.main:58 - Status: 404 HTTP Exception: Post not found
2025-06-14 12:31:20.037Z | (527bf3d6) | INFO     | uvicorn.access:476 - 127.0.0.1:2995 - "GET /posts/10 HTTP/1.1" 404
2025-06-14 12:31:29.756Z | (d562bb25) | INFO     | uvicorn.access:476 - 127.0.0.1:3003 - "GET /posts/10/ HTTP/1.1" 307
2025-06-14 12:31:29.759Z | (fe0dcabc) | ERROR    | storeapi.main:58 - Status: 404 HTTP Exception: Post not found
2025-06-14 12:31:29.761Z | (fe0dcabc) | INFO     | uvicorn.access:476 - 127.0.0.1:3003 - "GET /posts/10 HTTP/1.1" 404
2025-06-14 12:31:33.215Z | (74a1dd70) | INFO     | uvicorn.access:476 - 127.0.0.1:3003 - "GET /posts/10/ HTTP/1.1" 307
2025-06-14 12:31:33.218Z | (ddbb066c) | ERROR    | storeapi.main:58 - Status: 404 HTTP Exception: Post not found
2025-06-14 12:31:33.220Z | (ddbb066c) | INFO     | uvicorn.access:476 - 127.0.0.1:3003 - "GET /posts/10 HTTP/1.1" 404
2025-06-14 12:31:41.411Z | (1efc4538) | ERROR    | storeapi.main:58 - Status: 404 HTTP Exception: Post not found
2025-06-14 12:31:41.413Z | (1efc4538) | INFO     | uvicorn.access:476 - 127.0.0.1:3022 - "GET /posts/10 HTTP/1.1" 404
2025-06-14 12:31:46.359Z | (8bc561d4) | INFO     | uvicorn.access:476 - 127.0.0.1:3030 - "GET /posts/10/ HTTP/1.1" 307
2025-06-14 12:31:46.362Z | (98fb5574) | ERROR    | storeapi.main:58 - Status: 404 HTTP Exception: Post not found
2025-06-14 12:31:46.363Z | (98fb5574) | INFO     | uvicorn.access:476 - 127.0.0.1:3030 - "GET /posts/10 HTTP/1.1" 404
2025-06-14 12:31:51.380Z | (198b9d30) | ERROR    | storeapi.main:58 - Status: 404 HTTP Exception: Post not found
2025-06-14 12:31:51.382Z | (198b9d30) | INFO     | uvicorn.access:476 - 127.0.0.1:3023 - "GET /posts/10 HTTP/1.1" 404
2025-06-14 12:32:07.144Z | (107a7b66) | ERROR    | storeapi.main:58 - Status: 404 HTTP Exception: Post not found
2025-06-14 12:32:07.145Z | (107a7b66) | INFO     | uvicorn.access:476 - 127.0.0.1:3024 - "GET /posts/10 HTTP/1.1" 404
2025-06-14 12:32:31.105Z | (93b47572) | INFO     | uvicorn.access:476 - 127.0.0.1:3058 - "GET /posts/10/ HTTP/1.1" 307
2025-06-14 12:32:31.108Z | (bc2356ef) | ERROR    | storeapi.main:58 - Status: 404 HTTP Exception: Post not found
2025-06-14 12:32:31.110Z | (bc2356ef) | INFO     | uvicorn.access:476 - 127.0.0.1:3058 - "GET /posts/10 HTTP/1.1" 404
2025-06-14 12:44:29.930Z | (-) | INFO     | uvicorn.error:263 - Shutting down
2025-06-14 12:44:30.033Z | (-) | INFO     | uvicorn.error:67 - Waiting for application shutdown.
2025-06-14 12:44:30.035Z | (-) | INFO     | uvicorn.error:76 - Application shutdown complete.
2025-06-14 12:44:30.036Z | (-) | INFO     | uvicorn.error:93 - Finished server process [11896]
{"asctime": "2025-06-14 12:44:38", "msecs": 645.0, "correlation_id": "-", "levelname": "INFO", "name": "storeapi.main", "lineno": 22, "message": "Starting application lifespan"}
{"asctime": "2025-06-14 12:44:38", "msecs": 648.0, "correlation_id": "-", "levelname": "DEBUG", "name": "storeapi.main", "lineno": 23, "message": "Connecting to the database"}
{"asctime": "2025-06-14 12:44:38", "msecs": 917.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 62, "message": "Application startup complete."}
{"asctime": "2025-06-14 12:44:38", "msecs": 919.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 215, "message": "Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)", "color_message": "Uvicorn running on \u001b[1m%s://%s:%d\u001b[0m (Press CTRL+C to quit)"}
{"asctime": "2025-06-14 12:44:43", "msecs": 346.0, "correlation_id": "1dde1205", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:3170 - \"GET /posts/10/ HTTP/1.1\" 307"}
{"asctime": "2025-06-14 12:44:43", "msecs": 351.0, "correlation_id": "0a7f7a81", "levelname": "ERROR", "name": "storeapi.main", "lineno": 58, "message": "Status: 404 HTTP Exception: Post not found"}
{"asctime": "2025-06-14 12:44:43", "msecs": 352.0, "correlation_id": "0a7f7a81", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:3170 - \"GET /posts/10 HTTP/1.1\" 404"}
{"asctime": "2025-06-14 12:44:58", "msecs": 587.0, "correlation_id": "ee92ebb0", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:3176 - \"POST /posts/ HTTP/1.1\" 201"}
{"asctime": "2025-06-14 12:45:25", "msecs": 541.0, "correlation_id": "b7306931", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:3185 - \"POST /posts/ HTTP/1.1\" 201"}
{"asctime": "2025-06-14 15:39:13", "msecs": 225.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 263, "message": "Shutting down"}
{"asctime": "2025-06-14 15:39:13", "msecs": 339.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 93, "message": "Finished server process [10636]", "color_message": "Finished server process [\u001b[36m%d\u001b[0m]"}
{"asctime": "2025-06-14 15:39:13", "msecs": 380.0, "correlation_id": "-", "levelname": "ERROR", "name": "uvicorn.error", "lineno": 134, "message": "Traceback (most recent call last):\n  File \"C:\\Users\\USER\\anaconda3\\Lib\\asyncio\\runners.py\", line 194, in run\n    return runner.run(main)\n           ^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\USER\\anaconda3\\Lib\\asyncio\\runners.py\", line 118, in run\n    return self._loop.run_until_complete(task)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\USER\\anaconda3\\Lib\\asyncio\\base_events.py\", line 674, in run_until_complete\n    self.run_forever()\n  File \"C:\\Users\\USER\\anaconda3\\Lib\\asyncio\\windows_events.py\", line 322, in run_forever\n    super().run_forever()\n  File \"C:\\Users\\USER\\anaconda3\\Lib\\asyncio\\base_events.py\", line 641, in run_forever\n    self._run_once()\n  File \"C:\\Users\\USER\\anaconda3\\Lib\\asyncio\\base_events.py\", line 1986, in _run_once\n    handle._run()\n  File \"C:\\Users\\USER\\anaconda3\\Lib\\asyncio\\events.py\", line 88, in _run\n    self._context.run(self._callback, *self._args)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\uvicorn\\server.py\", line 69, in serve\n    with self.capture_signals():\n         ^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\USER\\anaconda3\\Lib\\contextlib.py\", line 144, in __exit__\n    next(self.gen)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\uvicorn\\server.py\", line 330, in capture_signals\n    signal.raise_signal(captured_signal)\n  File \"C:\\Users\\USER\\anaconda3\\Lib\\asyncio\\runners.py\", line 157, in _on_sigint\n    raise KeyboardInterrupt()\nKeyboardInterrupt\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 699, in lifespan\n    await receive()\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\uvicorn\\lifespan\\on.py\", line 137, in receive\n    return await self.receive_queue.get()\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\USER\\anaconda3\\Lib\\asyncio\\queues.py\", line 158, in get\n    await getter\nasyncio.exceptions.CancelledError\n"}
{"asctime": "2025-06-14 21:54:23", "msecs": 337.0, "correlation_id": "-", "levelname": "INFO", "name": "storeapi.main", "lineno": 23, "message": "Starting application lifespan"}
{"asctime": "2025-06-14 21:54:23", "msecs": 340.0, "correlation_id": "-", "levelname": "DEBUG", "name": "storeapi.main", "lineno": 24, "message": "Connecting to the database"}
{"asctime": "2025-06-14 21:54:23", "msecs": 618.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 62, "message": "Application startup complete."}
{"asctime": "2025-06-14 21:54:23", "msecs": 620.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 215, "message": "Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)", "color_message": "Uvicorn running on \u001b[1m%s://%s:%d\u001b[0m (Press CTRL+C to quit)"}
{"asctime": "2025-06-14 21:56:45", "msecs": 175.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 263, "message": "Shutting down"}
{"asctime": "2025-06-14 21:56:45", "msecs": 278.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 67, "message": "Waiting for application shutdown."}
{"asctime": "2025-06-14 21:56:45", "msecs": 283.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 76, "message": "Application shutdown complete."}
{"asctime": "2025-06-14 21:56:45", "msecs": 285.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 93, "message": "Finished server process [26576]", "color_message": "Finished server process [\u001b[36m%d\u001b[0m]"}
{"asctime": "2025-06-14 21:56:48", "msecs": 162.0, "correlation_id": "-", "levelname": "INFO", "name": "storeapi.main", "lineno": 23, "message": "Starting application lifespan"}
{"asctime": "2025-06-14 21:56:48", "msecs": 166.0, "correlation_id": "-", "levelname": "DEBUG", "name": "storeapi.main", "lineno": 24, "message": "Connecting to the database"}
{"asctime": "2025-06-14 21:56:48", "msecs": 437.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 62, "message": "Application startup complete."}
{"asctime": "2025-06-14 21:56:48", "msecs": 438.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 215, "message": "Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)", "color_message": "Uvicorn running on \u001b[1m%s://%s:%d\u001b[0m (Press CTRL+C to quit)"}
{"asctime": "2025-06-14 21:56:58", "msecs": 708.0, "correlation_id": "ef1c0449", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:9281 - \"GET /docs HTTP/1.1\" 200"}
{"asctime": "2025-06-14 21:56:59", "msecs": 333.0, "correlation_id": "1e2c96ac", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:9281 - \"GET /openapi.json HTTP/1.1\" 200"}
{"asctime": "2025-06-14 21:57:29", "msecs": 617.0, "correlation_id": "fb4747b1", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:9282 - \"POST /users/register HTTP/1.1\" 500"}
{"asctime": "2025-06-14 21:57:29", "msecs": 619.0, "correlation_id": "fb4747b1", "levelname": "ERROR", "name": "uvicorn.error", "lineno": 414, "message": "Exception in ASGI application\n", "exc_info": "Traceback (most recent call last):\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\uvicorn\\protocols\\http\\httptools_impl.py\", line 409, in run_asgi\n    result = await app(  # type: ignore[func-returns-value]\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\uvicorn\\middleware\\proxy_headers.py\", line 60, in __call__\n    return await self.app(scope, receive, send)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\fastapi\\applications.py\", line 1054, in __call__\n    await super().__call__(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\applications.py\", line 112, in __call__\n    await self.middleware_stack(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\middleware\\errors.py\", line 187, in __call__\n    raise exc\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\middleware\\errors.py\", line 165, in __call__\n    await self.app(scope, receive, _send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\asgi_correlation_id\\middleware.py\", line 90, in __call__\n    await self.app(scope, receive, handle_outgoing_request)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\middleware\\cors.py\", line 93, in __call__\n    await self.simple_response(scope, receive, send, request_headers=headers)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\middleware\\cors.py\", line 144, in simple_response\n    await self.app(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\middleware\\exceptions.py\", line 62, in __call__\n    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 53, in wrapped_app\n    raise exc\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 42, in wrapped_app\n    await app(scope, receive, sender)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 714, in __call__\n    await self.middleware_stack(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 734, in app\n    await route.handle(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 288, in handle\n    await self.app(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 76, in app\n    await wrap_app_handling_exceptions(app, request)(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 53, in wrapped_app\n    raise exc\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 42, in wrapped_app\n    await app(scope, receive, sender)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 73, in app\n    response = await f(request)\n               ^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\fastapi\\routing.py\", line 301, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\fastapi\\routing.py\", line 212, in run_endpoint_function\n    return await dependant.call(**values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\storeapi\\controller\\user.py\", line 36, in register_user\n    user_id = await database.execute(query)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\databases\\core.py\", line 199, in execute\n    return await connection.execute(query, values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\databases\\core.py\", line 318, in execute\n    return await self._connection.execute(built_query)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\databases\\backends\\postgres.py\", line 140, in execute\n    query_str, args, _ = self._compile(query)\n                         ^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\databases\\backends\\postgres.py\", line 165, in _compile\n    compiled = query.compile(\n               ^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\sqlalchemy\\sql\\elements.py\", line 308, in compile\n    return self._compiler(dialect, **kw)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\sqlalchemy\\sql\\elements.py\", line 320, in _compiler\n    return dialect.statement_compiler(dialect, self, **kw)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\sqlalchemy\\sql\\compiler.py\", line 1446, in __init__\n    Compiled.__init__(self, dialect, statement, **kwargs)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\sqlalchemy\\sql\\compiler.py\", line 886, in __init__\n    self.string = self.process(self.statement, **compile_kwargs)\n                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\sqlalchemy\\sql\\compiler.py\", line 932, in process\n    return obj._compiler_dispatch(self, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\sqlalchemy\\sql\\visitors.py\", line 141, in _compiler_dispatch\n    return meth(self, **kw)  # type: ignore  # noqa: E501\n           ^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\sqlalchemy\\sql\\compiler.py\", line 5843, in visit_insert\n    crud_params_struct = crud._get_crud_params(\n                         ^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\sqlalchemy\\sql\\crud.py\", line 335, in _get_crud_params\n    raise exc.CompileError(\nsqlalchemy.exc.CompileError: Unconsumed column names: password"}
{"asctime": "2025-06-14 21:57:39", "msecs": 647.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 263, "message": "Shutting down"}
{"asctime": "2025-06-14 21:57:39", "msecs": 749.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 67, "message": "Waiting for application shutdown."}
{"asctime": "2025-06-14 21:57:39", "msecs": 754.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 76, "message": "Application shutdown complete."}
{"asctime": "2025-06-14 21:57:39", "msecs": 755.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 93, "message": "Finished server process [27832]", "color_message": "Finished server process [\u001b[36m%d\u001b[0m]"}
{"asctime": "2025-06-14 21:58:59", "msecs": 646.0, "correlation_id": "-", "levelname": "INFO", "name": "storeapi.main", "lineno": 23, "message": "Starting application lifespan"}
{"asctime": "2025-06-14 21:58:59", "msecs": 650.0, "correlation_id": "-", "levelname": "DEBUG", "name": "storeapi.main", "lineno": 24, "message": "Connecting to the database"}
{"asctime": "2025-06-14 21:58:59", "msecs": 918.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 62, "message": "Application startup complete."}
{"asctime": "2025-06-14 21:58:59", "msecs": 920.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 215, "message": "Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)", "color_message": "Uvicorn running on \u001b[1m%s://%s:%d\u001b[0m (Press CTRL+C to quit)"}
{"asctime": "2025-06-14 21:59:06", "msecs": 354.0, "correlation_id": "db523f03", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:9368 - \"POST /users/register HTTP/1.1\" 500"}
{"asctime": "2025-06-14 21:59:06", "msecs": 356.0, "correlation_id": "db523f03", "levelname": "ERROR", "name": "uvicorn.error", "lineno": 414, "message": "Exception in ASGI application\n", "exc_info": "Traceback (most recent call last):\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\uvicorn\\protocols\\http\\httptools_impl.py\", line 409, in run_asgi\n    result = await app(  # type: ignore[func-returns-value]\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\uvicorn\\middleware\\proxy_headers.py\", line 60, in __call__\n    return await self.app(scope, receive, send)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\fastapi\\applications.py\", line 1054, in __call__\n    await super().__call__(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\applications.py\", line 112, in __call__\n    await self.middleware_stack(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\middleware\\errors.py\", line 187, in __call__\n    raise exc\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\middleware\\errors.py\", line 165, in __call__\n    await self.app(scope, receive, _send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\asgi_correlation_id\\middleware.py\", line 90, in __call__\n    await self.app(scope, receive, handle_outgoing_request)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\middleware\\cors.py\", line 93, in __call__\n    await self.simple_response(scope, receive, send, request_headers=headers)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\middleware\\cors.py\", line 144, in simple_response\n    await self.app(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\middleware\\exceptions.py\", line 62, in __call__\n    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 53, in wrapped_app\n    raise exc\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 42, in wrapped_app\n    await app(scope, receive, sender)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 714, in __call__\n    await self.middleware_stack(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 734, in app\n    await route.handle(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 288, in handle\n    await self.app(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 76, in app\n    await wrap_app_handling_exceptions(app, request)(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 53, in wrapped_app\n    raise exc\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 42, in wrapped_app\n    await app(scope, receive, sender)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 73, in app\n    response = await f(request)\n               ^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\fastapi\\routing.py\", line 301, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\fastapi\\routing.py\", line 212, in run_endpoint_function\n    return await dependant.call(**values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\storeapi\\controller\\user.py\", line 36, in register_user\n    user_id = await database.execute(query)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\databases\\core.py\", line 199, in execute\n    return await connection.execute(query, values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\databases\\core.py\", line 318, in execute\n    return await self._connection.execute(built_query)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\databases\\backends\\postgres.py\", line 140, in execute\n    query_str, args, _ = self._compile(query)\n                         ^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\databases\\backends\\postgres.py\", line 165, in _compile\n    compiled = query.compile(\n               ^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\sqlalchemy\\sql\\elements.py\", line 308, in compile\n    return self._compiler(dialect, **kw)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\sqlalchemy\\sql\\elements.py\", line 320, in _compiler\n    return dialect.statement_compiler(dialect, self, **kw)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\sqlalchemy\\sql\\compiler.py\", line 1446, in __init__\n    Compiled.__init__(self, dialect, statement, **kwargs)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\sqlalchemy\\sql\\compiler.py\", line 886, in __init__\n    self.string = self.process(self.statement, **compile_kwargs)\n                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\sqlalchemy\\sql\\compiler.py\", line 932, in process\n    return obj._compiler_dispatch(self, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\sqlalchemy\\sql\\visitors.py\", line 141, in _compiler_dispatch\n    return meth(self, **kw)  # type: ignore  # noqa: E501\n           ^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\sqlalchemy\\sql\\compiler.py\", line 5843, in visit_insert\n    crud_params_struct = crud._get_crud_params(\n                         ^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\sqlalchemy\\sql\\crud.py\", line 335, in _get_crud_params\n    raise exc.CompileError(\nsqlalchemy.exc.CompileError: Unconsumed column names: password"}
{"asctime": "2025-06-14 21:59:11", "msecs": 420.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 263, "message": "Shutting down"}
{"asctime": "2025-06-14 21:59:11", "msecs": 523.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 67, "message": "Waiting for application shutdown."}
{"asctime": "2025-06-14 21:59:11", "msecs": 528.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 76, "message": "Application shutdown complete."}
{"asctime": "2025-06-14 21:59:11", "msecs": 530.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 93, "message": "Finished server process [24064]", "color_message": "Finished server process [\u001b[36m%d\u001b[0m]"}
{"asctime": "2025-06-14 21:59:41", "msecs": 691.0, "correlation_id": "-", "levelname": "INFO", "name": "storeapi.main", "lineno": 23, "message": "Starting application lifespan"}
{"asctime": "2025-06-14 21:59:41", "msecs": 694.0, "correlation_id": "-", "levelname": "DEBUG", "name": "storeapi.main", "lineno": 24, "message": "Connecting to the database"}
{"asctime": "2025-06-14 21:59:41", "msecs": 964.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 62, "message": "Application startup complete."}
{"asctime": "2025-06-14 21:59:41", "msecs": 965.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 215, "message": "Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)", "color_message": "Uvicorn running on \u001b[1m%s://%s:%d\u001b[0m (Press CTRL+C to quit)"}
{"asctime": "2025-06-14 21:59:47", "msecs": 633.0, "correlation_id": "02cc6270", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:9405 - \"POST /users/register HTTP/1.1\" 500"}
{"asctime": "2025-06-14 21:59:47", "msecs": 635.0, "correlation_id": "02cc6270", "levelname": "ERROR", "name": "uvicorn.error", "lineno": 414, "message": "Exception in ASGI application\n", "exc_info": "Traceback (most recent call last):\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\uvicorn\\protocols\\http\\httptools_impl.py\", line 409, in run_asgi\n    result = await app(  # type: ignore[func-returns-value]\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\uvicorn\\middleware\\proxy_headers.py\", line 60, in __call__\n    return await self.app(scope, receive, send)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\fastapi\\applications.py\", line 1054, in __call__\n    await super().__call__(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\applications.py\", line 112, in __call__\n    await self.middleware_stack(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\middleware\\errors.py\", line 187, in __call__\n    raise exc\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\middleware\\errors.py\", line 165, in __call__\n    await self.app(scope, receive, _send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\asgi_correlation_id\\middleware.py\", line 90, in __call__\n    await self.app(scope, receive, handle_outgoing_request)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\middleware\\cors.py\", line 93, in __call__\n    await self.simple_response(scope, receive, send, request_headers=headers)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\middleware\\cors.py\", line 144, in simple_response\n    await self.app(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\middleware\\exceptions.py\", line 62, in __call__\n    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 53, in wrapped_app\n    raise exc\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 42, in wrapped_app\n    await app(scope, receive, sender)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 714, in __call__\n    await self.middleware_stack(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 734, in app\n    await route.handle(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 288, in handle\n    await self.app(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 76, in app\n    await wrap_app_handling_exceptions(app, request)(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 53, in wrapped_app\n    raise exc\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 42, in wrapped_app\n    await app(scope, receive, sender)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 73, in app\n    response = await f(request)\n               ^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\fastapi\\routing.py\", line 301, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\fastapi\\routing.py\", line 212, in run_endpoint_function\n    return await dependant.call(**values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\storeapi\\controller\\user.py\", line 36, in register_user\n    user_id = await database.execute(query)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\databases\\core.py\", line 199, in execute\n    return await connection.execute(query, values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\databases\\core.py\", line 318, in execute\n    return await self._connection.execute(built_query)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\databases\\backends\\postgres.py\", line 140, in execute\n    query_str, args, _ = self._compile(query)\n                         ^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\databases\\backends\\postgres.py\", line 165, in _compile\n    compiled = query.compile(\n               ^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\sqlalchemy\\sql\\elements.py\", line 308, in compile\n    return self._compiler(dialect, **kw)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\sqlalchemy\\sql\\elements.py\", line 320, in _compiler\n    return dialect.statement_compiler(dialect, self, **kw)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\sqlalchemy\\sql\\compiler.py\", line 1446, in __init__\n    Compiled.__init__(self, dialect, statement, **kwargs)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\sqlalchemy\\sql\\compiler.py\", line 886, in __init__\n    self.string = self.process(self.statement, **compile_kwargs)\n                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\sqlalchemy\\sql\\compiler.py\", line 932, in process\n    return obj._compiler_dispatch(self, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\sqlalchemy\\sql\\visitors.py\", line 141, in _compiler_dispatch\n    return meth(self, **kw)  # type: ignore  # noqa: E501\n           ^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\sqlalchemy\\sql\\compiler.py\", line 5843, in visit_insert\n    crud_params_struct = crud._get_crud_params(\n                         ^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\sqlalchemy\\sql\\crud.py\", line 335, in _get_crud_params\n    raise exc.CompileError(\nsqlalchemy.exc.CompileError: Unconsumed column names: password"}
{"asctime": "2025-06-14 22:02:49", "msecs": 26.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 263, "message": "Shutting down"}
{"asctime": "2025-06-14 22:02:49", "msecs": 124.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 93, "message": "Finished server process [24808]", "color_message": "Finished server process [\u001b[36m%d\u001b[0m]"}
{"asctime": "2025-06-14 22:02:49", "msecs": 141.0, "correlation_id": "-", "levelname": "ERROR", "name": "uvicorn.error", "lineno": 134, "message": "Traceback (most recent call last):\n  File \"C:\\Users\\USER\\anaconda3\\Lib\\asyncio\\runners.py\", line 194, in run\n    return runner.run(main)\n           ^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\USER\\anaconda3\\Lib\\asyncio\\runners.py\", line 118, in run\n    return self._loop.run_until_complete(task)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\USER\\anaconda3\\Lib\\asyncio\\base_events.py\", line 674, in run_until_complete\n    self.run_forever()\n  File \"C:\\Users\\USER\\anaconda3\\Lib\\asyncio\\windows_events.py\", line 322, in run_forever\n    super().run_forever()\n  File \"C:\\Users\\USER\\anaconda3\\Lib\\asyncio\\base_events.py\", line 641, in run_forever\n    self._run_once()\n  File \"C:\\Users\\USER\\anaconda3\\Lib\\asyncio\\base_events.py\", line 1986, in _run_once\n    handle._run()\n  File \"C:\\Users\\USER\\anaconda3\\Lib\\asyncio\\events.py\", line 88, in _run\n    self._context.run(self._callback, *self._args)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\uvicorn\\server.py\", line 69, in serve\n    with self.capture_signals():\n         ^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\USER\\anaconda3\\Lib\\contextlib.py\", line 144, in __exit__\n    next(self.gen)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\uvicorn\\server.py\", line 330, in capture_signals\n    signal.raise_signal(captured_signal)\n  File \"C:\\Users\\USER\\anaconda3\\Lib\\asyncio\\runners.py\", line 157, in _on_sigint\n    raise KeyboardInterrupt()\nKeyboardInterrupt\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 699, in lifespan\n    await receive()\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\uvicorn\\lifespan\\on.py\", line 137, in receive\n    return await self.receive_queue.get()\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\USER\\anaconda3\\Lib\\asyncio\\queues.py\", line 158, in get\n    await getter\nasyncio.exceptions.CancelledError\n"}
{"asctime": "2025-06-14 22:02:51", "msecs": 666.0, "correlation_id": "-", "levelname": "INFO", "name": "storeapi.main", "lineno": 23, "message": "Starting application lifespan"}
{"asctime": "2025-06-14 22:02:51", "msecs": 669.0, "correlation_id": "-", "levelname": "DEBUG", "name": "storeapi.main", "lineno": 24, "message": "Connecting to the database"}
{"asctime": "2025-06-14 22:02:51", "msecs": 946.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 62, "message": "Application startup complete."}
{"asctime": "2025-06-14 22:02:51", "msecs": 947.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 215, "message": "Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)", "color_message": "Uvicorn running on \u001b[1m%s://%s:%d\u001b[0m (Press CTRL+C to quit)"}
{"asctime": "2025-06-14 22:02:55", "msecs": 465.0, "correlation_id": "269614da", "levelname": "INFO", "name": "storeapi.controller.user", "lineno": 32, "message": "Registering user: rafi"}
{"asctime": "2025-06-14 22:02:55", "msecs": 671.0, "correlation_id": "269614da", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:9718 - \"POST /users/register HTTP/1.1\" 500"}
{"asctime": "2025-06-14 22:02:55", "msecs": 672.0, "correlation_id": "269614da", "levelname": "ERROR", "name": "uvicorn.error", "lineno": 414, "message": "Exception in ASGI application\n", "exc_info": "Traceback (most recent call last):\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\uvicorn\\protocols\\http\\httptools_impl.py\", line 409, in run_asgi\n    result = await app(  # type: ignore[func-returns-value]\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\uvicorn\\middleware\\proxy_headers.py\", line 60, in __call__\n    return await self.app(scope, receive, send)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\fastapi\\applications.py\", line 1054, in __call__\n    await super().__call__(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\applications.py\", line 112, in __call__\n    await self.middleware_stack(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\middleware\\errors.py\", line 187, in __call__\n    raise exc\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\middleware\\errors.py\", line 165, in __call__\n    await self.app(scope, receive, _send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\asgi_correlation_id\\middleware.py\", line 90, in __call__\n    await self.app(scope, receive, handle_outgoing_request)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\middleware\\cors.py\", line 93, in __call__\n    await self.simple_response(scope, receive, send, request_headers=headers)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\middleware\\cors.py\", line 144, in simple_response\n    await self.app(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\middleware\\exceptions.py\", line 62, in __call__\n    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 53, in wrapped_app\n    raise exc\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 42, in wrapped_app\n    await app(scope, receive, sender)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 714, in __call__\n    await self.middleware_stack(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 734, in app\n    await route.handle(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 288, in handle\n    await self.app(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 76, in app\n    await wrap_app_handling_exceptions(app, request)(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 53, in wrapped_app\n    raise exc\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 42, in wrapped_app\n    await app(scope, receive, sender)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 73, in app\n    response = await f(request)\n               ^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\fastapi\\routing.py\", line 301, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\fastapi\\routing.py\", line 212, in run_endpoint_function\n    return await dependant.call(**values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\storeapi\\controller\\user.py\", line 41, in register_user\n    logger.debug(f\"Executing query: {query}\")\n                                    ^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\sqlalchemy\\sql\\elements.py\", line 323, in __str__\n    return str(self.compile())\n               ^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\sqlalchemy\\sql\\elements.py\", line 308, in compile\n    return self._compiler(dialect, **kw)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\sqlalchemy\\sql\\elements.py\", line 320, in _compiler\n    return dialect.statement_compiler(dialect, self, **kw)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\sqlalchemy\\sql\\compiler.py\", line 1446, in __init__\n    Compiled.__init__(self, dialect, statement, **kwargs)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\sqlalchemy\\sql\\compiler.py\", line 886, in __init__\n    self.string = self.process(self.statement, **compile_kwargs)\n                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\sqlalchemy\\sql\\compiler.py\", line 932, in process\n    return obj._compiler_dispatch(self, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\sqlalchemy\\sql\\visitors.py\", line 141, in _compiler_dispatch\n    return meth(self, **kw)  # type: ignore  # noqa: E501\n           ^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\sqlalchemy\\sql\\compiler.py\", line 5843, in visit_insert\n    crud_params_struct = crud._get_crud_params(\n                         ^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\sqlalchemy\\sql\\crud.py\", line 335, in _get_crud_params\n    raise exc.CompileError(\nsqlalchemy.exc.CompileError: Unconsumed column names: password"}
{"asctime": "2025-06-14 22:04:09", "msecs": 355.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 263, "message": "Shutting down"}
{"asctime": "2025-06-14 22:04:09", "msecs": 458.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 67, "message": "Waiting for application shutdown."}
{"asctime": "2025-06-14 22:04:09", "msecs": 462.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 76, "message": "Application shutdown complete."}
{"asctime": "2025-06-14 22:04:09", "msecs": 463.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 93, "message": "Finished server process [12960]", "color_message": "Finished server process [\u001b[36m%d\u001b[0m]"}
{"asctime": "2025-06-14 22:04:13", "msecs": 513.0, "correlation_id": "-", "levelname": "INFO", "name": "storeapi.main", "lineno": 23, "message": "Starting application lifespan"}
{"asctime": "2025-06-14 22:04:13", "msecs": 517.0, "correlation_id": "-", "levelname": "DEBUG", "name": "storeapi.main", "lineno": 24, "message": "Connecting to the database"}
{"asctime": "2025-06-14 22:04:13", "msecs": 790.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 62, "message": "Application startup complete."}
{"asctime": "2025-06-14 22:04:13", "msecs": 791.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 215, "message": "Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)", "color_message": "Uvicorn running on \u001b[1m%s://%s:%d\u001b[0m (Press CTRL+C to quit)"}
{"asctime": "2025-06-14 22:04:29", "msecs": 32.0, "correlation_id": "6a8cc3e0", "levelname": "INFO", "name": "storeapi.controller.user", "lineno": 32, "message": "Registering user: rafi"}
{"asctime": "2025-06-14 22:04:29", "msecs": 244.0, "correlation_id": "6a8cc3e0", "levelname": "DEBUG", "name": "storeapi.controller.user", "lineno": 41, "message": "Executing query: INSERT INTO users (username, email, hashed_password) VALUES (:username, :email, :hashed_password)"}
{"asctime": "2025-06-14 22:04:29", "msecs": 250.0, "correlation_id": "6a8cc3e0", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:9795 - \"POST /users/register HTTP/1.1\" 201"}
{"asctime": "2025-06-14 22:06:09", "msecs": 303.0, "correlation_id": "1e8c4362", "levelname": "ERROR", "name": "storeapi.main", "lineno": 60, "message": "Status: 401 HTTP Exception: Not authenticated"}
{"asctime": "2025-06-14 22:06:09", "msecs": 305.0, "correlation_id": "1e8c4362", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:10047 - \"POST /posts/ HTTP/1.1\" 401"}
{"asctime": "2025-06-14 22:06:34", "msecs": 627.0, "correlation_id": "0217b92b", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:10048 - \"POST /users/login HTTP/1.1\" 405"}
{"asctime": "2025-06-14 22:06:41", "msecs": 114.0, "correlation_id": "ca92fe8c", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:10065 - \"POST /users/login HTTP/1.1\" 405"}
{"asctime": "2025-06-14 22:07:18", "msecs": 420.0, "correlation_id": "ed39da95", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:10066 - \"GET /docs HTTP/1.1\" 200"}
{"asctime": "2025-06-14 22:07:18", "msecs": 564.0, "correlation_id": "d1b31ba1", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:10066 - \"GET /openapi.json HTTP/1.1\" 200"}
{"asctime": "2025-06-14 22:07:43", "msecs": 541.0, "correlation_id": "99dac842", "levelname": "ERROR", "name": "storeapi.main", "lineno": 60, "message": "Status: 401 HTTP Exception: Not authenticated"}
{"asctime": "2025-06-14 22:07:43", "msecs": 542.0, "correlation_id": "99dac842", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:10108 - \"POST /posts/ HTTP/1.1\" 401"}
{"asctime": "2025-06-14 22:07:59", "msecs": 127.0, "correlation_id": "51949bf4", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:10091 - \"GET /users/login?username=rafi&password=12345%40 HTTP/1.1\" 500"}
{"asctime": "2025-06-14 22:07:59", "msecs": 130.0, "correlation_id": "51949bf4", "levelname": "ERROR", "name": "uvicorn.error", "lineno": 414, "message": "Exception in ASGI application\n", "exc_info": "Traceback (most recent call last):\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\uvicorn\\protocols\\http\\httptools_impl.py\", line 409, in run_asgi\n    result = await app(  # type: ignore[func-returns-value]\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\uvicorn\\middleware\\proxy_headers.py\", line 60, in __call__\n    return await self.app(scope, receive, send)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\fastapi\\applications.py\", line 1054, in __call__\n    await super().__call__(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\applications.py\", line 112, in __call__\n    await self.middleware_stack(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\middleware\\errors.py\", line 187, in __call__\n    raise exc\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\middleware\\errors.py\", line 165, in __call__\n    await self.app(scope, receive, _send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\asgi_correlation_id\\middleware.py\", line 90, in __call__\n    await self.app(scope, receive, handle_outgoing_request)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\middleware\\cors.py\", line 85, in __call__\n    await self.app(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\middleware\\exceptions.py\", line 62, in __call__\n    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 53, in wrapped_app\n    raise exc\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 42, in wrapped_app\n    await app(scope, receive, sender)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 714, in __call__\n    await self.middleware_stack(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 734, in app\n    await route.handle(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 288, in handle\n    await self.app(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 76, in app\n    await wrap_app_handling_exceptions(app, request)(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 53, in wrapped_app\n    raise exc\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 42, in wrapped_app\n    await app(scope, receive, sender)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 73, in app\n    response = await f(request)\n               ^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\fastapi\\routing.py\", line 301, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\fastapi\\routing.py\", line 212, in run_endpoint_function\n    return await dependant.call(**values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\storeapi\\controller\\user.py\", line 65, in login\n    \"id\": user[\"id\"],\n          ~~~~^^^^^^\nTypeError: 'coroutine' object is not subscriptable"}
{"asctime": "2025-06-14 22:08:21", "msecs": 16.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 263, "message": "Shutting down"}
{"asctime": "2025-06-14 22:08:21", "msecs": 119.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 67, "message": "Waiting for application shutdown."}
{"asctime": "2025-06-14 22:08:21", "msecs": 124.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 76, "message": "Application shutdown complete."}
{"asctime": "2025-06-14 22:08:21", "msecs": 125.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 93, "message": "Finished server process [24324]", "color_message": "Finished server process [\u001b[36m%d\u001b[0m]"}
{"asctime": "2025-06-14 22:08:32", "msecs": 321.0, "correlation_id": "-", "levelname": "INFO", "name": "storeapi.main", "lineno": 23, "message": "Starting application lifespan"}
{"asctime": "2025-06-14 22:08:32", "msecs": 324.0, "correlation_id": "-", "levelname": "DEBUG", "name": "storeapi.main", "lineno": 24, "message": "Connecting to the database"}
{"asctime": "2025-06-14 22:08:32", "msecs": 591.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 62, "message": "Application startup complete."}
{"asctime": "2025-06-14 22:08:32", "msecs": 592.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 215, "message": "Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)", "color_message": "Uvicorn running on \u001b[1m%s://%s:%d\u001b[0m (Press CTRL+C to quit)"}
{"asctime": "2025-06-14 22:08:41", "msecs": 891.0, "correlation_id": "831529f4", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:10171 - \"GET /users/login?username=rafi&password=12345%40 HTTP/1.1\" 500"}
{"asctime": "2025-06-14 22:08:41", "msecs": 893.0, "correlation_id": "831529f4", "levelname": "ERROR", "name": "uvicorn.error", "lineno": 414, "message": "Exception in ASGI application\n", "exc_info": "Traceback (most recent call last):\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\uvicorn\\protocols\\http\\httptools_impl.py\", line 409, in run_asgi\n    result = await app(  # type: ignore[func-returns-value]\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\uvicorn\\middleware\\proxy_headers.py\", line 60, in __call__\n    return await self.app(scope, receive, send)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\fastapi\\applications.py\", line 1054, in __call__\n    await super().__call__(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\applications.py\", line 112, in __call__\n    await self.middleware_stack(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\middleware\\errors.py\", line 187, in __call__\n    raise exc\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\middleware\\errors.py\", line 165, in __call__\n    await self.app(scope, receive, _send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\asgi_correlation_id\\middleware.py\", line 90, in __call__\n    await self.app(scope, receive, handle_outgoing_request)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\middleware\\cors.py\", line 85, in __call__\n    await self.app(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\middleware\\exceptions.py\", line 62, in __call__\n    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 53, in wrapped_app\n    raise exc\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 42, in wrapped_app\n    await app(scope, receive, sender)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 714, in __call__\n    await self.middleware_stack(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 734, in app\n    await route.handle(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 288, in handle\n    await self.app(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 76, in app\n    await wrap_app_handling_exceptions(app, request)(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 53, in wrapped_app\n    raise exc\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 42, in wrapped_app\n    await app(scope, receive, sender)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 73, in app\n    response = await f(request)\n               ^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\fastapi\\routing.py\", line 301, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\fastapi\\routing.py\", line 212, in run_endpoint_function\n    return await dependant.call(**values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\storeapi\\controller\\user.py\", line 61, in login\n    return JSONResponse(\n           ^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\responses.py\", line 182, in __init__\n    super().__init__(content, status_code, headers, media_type, background)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\responses.py\", line 45, in __init__\n    self.body = self.render(content)\n                ^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\responses.py\", line 185, in render\n    return json.dumps(\n           ^^^^^^^^^^^\n  File \"C:\\Users\\USER\\anaconda3\\Lib\\json\\__init__.py\", line 238, in dumps\n    **kw).encode(obj)\n          ^^^^^^^^^^^\n  File \"C:\\Users\\USER\\anaconda3\\Lib\\json\\encoder.py\", line 200, in encode\n    chunks = self.iterencode(o, _one_shot=True)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\USER\\anaconda3\\Lib\\json\\encoder.py\", line 258, in iterencode\n    return _iterencode(o, 0)\n           ^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\USER\\anaconda3\\Lib\\json\\encoder.py\", line 180, in default\n    raise TypeError(f'Object of type {o.__class__.__name__} '\nTypeError: Object of type coroutine is not JSON serializable"}
{"asctime": "2025-06-14 22:09:11", "msecs": 192.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 263, "message": "Shutting down"}
{"asctime": "2025-06-14 22:09:11", "msecs": 295.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 67, "message": "Waiting for application shutdown."}
{"asctime": "2025-06-14 22:09:11", "msecs": 299.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 76, "message": "Application shutdown complete."}
{"asctime": "2025-06-14 22:09:11", "msecs": 301.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 93, "message": "Finished server process [25520]", "color_message": "Finished server process [\u001b[36m%d\u001b[0m]"}
{"asctime": "2025-06-14 22:09:13", "msecs": 780.0, "correlation_id": "-", "levelname": "INFO", "name": "storeapi.main", "lineno": 23, "message": "Starting application lifespan"}
{"asctime": "2025-06-14 22:09:13", "msecs": 783.0, "correlation_id": "-", "levelname": "DEBUG", "name": "storeapi.main", "lineno": 24, "message": "Connecting to the database"}
{"asctime": "2025-06-14 22:09:14", "msecs": 54.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 62, "message": "Application startup complete."}
{"asctime": "2025-06-14 22:09:14", "msecs": 56.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 215, "message": "Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)", "color_message": "Uvicorn running on \u001b[1m%s://%s:%d\u001b[0m (Press CTRL+C to quit)"}
{"asctime": "2025-06-14 22:09:26", "msecs": 296.0, "correlation_id": "a8d7f362", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:10211 - \"GET /users/login?username=rafi&password=12345%40 HTTP/1.1\" 200"}
{"asctime": "2025-06-14 22:10:30", "msecs": 668.0, "correlation_id": "08563213", "levelname": "ERROR", "name": "storeapi.main", "lineno": 60, "message": "Status: 401 HTTP Exception: Could not validate credentials"}
{"asctime": "2025-06-14 22:10:30", "msecs": 669.0, "correlation_id": "08563213", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:10285 - \"POST /posts/ HTTP/1.1\" 401"}
{"asctime": "2025-06-14 22:12:53", "msecs": 431.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 263, "message": "Shutting down"}
{"asctime": "2025-06-14 22:12:53", "msecs": 534.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 67, "message": "Waiting for application shutdown."}
{"asctime": "2025-06-14 22:12:53", "msecs": 539.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 76, "message": "Application shutdown complete."}
{"asctime": "2025-06-14 22:12:53", "msecs": 540.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 93, "message": "Finished server process [24112]", "color_message": "Finished server process [\u001b[36m%d\u001b[0m]"}
{"asctime": "2025-06-14 22:12:56", "msecs": 151.0, "correlation_id": "-", "levelname": "INFO", "name": "storeapi.main", "lineno": 23, "message": "Starting application lifespan"}
{"asctime": "2025-06-14 22:12:56", "msecs": 154.0, "correlation_id": "-", "levelname": "DEBUG", "name": "storeapi.main", "lineno": 24, "message": "Connecting to the database"}
{"asctime": "2025-06-14 22:12:56", "msecs": 418.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 62, "message": "Application startup complete."}
{"asctime": "2025-06-14 22:12:56", "msecs": 419.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 215, "message": "Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)", "color_message": "Uvicorn running on \u001b[1m%s://%s:%d\u001b[0m (Press CTRL+C to quit)"}
{"asctime": "2025-06-14 22:13:00", "msecs": 355.0, "correlation_id": "3962301a", "levelname": "ERROR", "name": "storeapi.main", "lineno": 60, "message": "Status: 401 HTTP Exception: Could not validate credentials"}
{"asctime": "2025-06-14 22:13:00", "msecs": 356.0, "correlation_id": "3962301a", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:10411 - \"POST /posts/ HTTP/1.1\" 401"}
{"asctime": "2025-06-14 22:13:26", "msecs": 550.0, "correlation_id": "9d80b17a", "levelname": "ERROR", "name": "storeapi.main", "lineno": 60, "message": "Status: 401 HTTP Exception: Could not validate credentials"}
{"asctime": "2025-06-14 22:13:26", "msecs": 551.0, "correlation_id": "9d80b17a", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:10423 - \"POST /posts/ HTTP/1.1\" 401"}
{"asctime": "2025-06-14 22:14:04", "msecs": 37.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 263, "message": "Shutting down"}
{"asctime": "2025-06-14 22:14:04", "msecs": 139.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 67, "message": "Waiting for application shutdown."}
{"asctime": "2025-06-14 22:14:04", "msecs": 144.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 76, "message": "Application shutdown complete."}
{"asctime": "2025-06-14 22:14:04", "msecs": 146.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 93, "message": "Finished server process [3060]", "color_message": "Finished server process [\u001b[36m%d\u001b[0m]"}
{"asctime": "2025-06-14 22:14:06", "msecs": 625.0, "correlation_id": "-", "levelname": "INFO", "name": "storeapi.main", "lineno": 23, "message": "Starting application lifespan"}
{"asctime": "2025-06-14 22:14:06", "msecs": 628.0, "correlation_id": "-", "levelname": "DEBUG", "name": "storeapi.main", "lineno": 24, "message": "Connecting to the database"}
{"asctime": "2025-06-14 22:14:06", "msecs": 891.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 62, "message": "Application startup complete."}
{"asctime": "2025-06-14 22:14:06", "msecs": 893.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 215, "message": "Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)", "color_message": "Uvicorn running on \u001b[1m%s://%s:%d\u001b[0m (Press CTRL+C to quit)"}
{"asctime": "2025-06-14 22:14:09", "msecs": 443.0, "correlation_id": "3bcf993a", "levelname": "ERROR", "name": "storeapi.main", "lineno": 60, "message": "Status: 401 HTTP Exception: Could not validate credentials"}
{"asctime": "2025-06-14 22:14:09", "msecs": 444.0, "correlation_id": "3bcf993a", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:10465 - \"POST /posts/ HTTP/1.1\" 401"}
{"asctime": "2025-06-14 22:14:16", "msecs": 600.0, "correlation_id": "a42d5000", "levelname": "ERROR", "name": "storeapi.main", "lineno": 60, "message": "Status: 401 HTTP Exception: Could not validate credentials"}
{"asctime": "2025-06-14 22:14:16", "msecs": 602.0, "correlation_id": "a42d5000", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:10473 - \"POST /posts/ HTTP/1.1\" 401"}
{"asctime": "2025-06-14 22:14:43", "msecs": 518.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 263, "message": "Shutting down"}
{"asctime": "2025-06-14 22:14:43", "msecs": 621.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 67, "message": "Waiting for application shutdown."}
{"asctime": "2025-06-14 22:14:43", "msecs": 624.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 76, "message": "Application shutdown complete."}
{"asctime": "2025-06-14 22:14:43", "msecs": 626.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 93, "message": "Finished server process [14516]", "color_message": "Finished server process [\u001b[36m%d\u001b[0m]"}
{"asctime": "2025-06-14 22:14:45", "msecs": 779.0, "correlation_id": "-", "levelname": "INFO", "name": "storeapi.main", "lineno": 23, "message": "Starting application lifespan"}
{"asctime": "2025-06-14 22:14:45", "msecs": 783.0, "correlation_id": "-", "levelname": "DEBUG", "name": "storeapi.main", "lineno": 24, "message": "Connecting to the database"}
{"asctime": "2025-06-14 22:14:46", "msecs": 58.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 62, "message": "Application startup complete."}
{"asctime": "2025-06-14 22:14:46", "msecs": 60.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 215, "message": "Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)", "color_message": "Uvicorn running on \u001b[1m%s://%s:%d\u001b[0m (Press CTRL+C to quit)"}
{"asctime": "2025-06-14 22:14:50", "msecs": 512.0, "correlation_id": "b8e8cf29", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 113, "message": "Received token: eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MSwic3ViIjoicmFmaSIsImVtYWlsIjoicmFmaS5oYXNhbkBnbWFpbC5jb20iLCJleHAiOjE3NDk5MTczNzF9.tj-YVG02Tr4RkysDuyCHcleSgqPTXekltkLNMLCt690"}
{"asctime": "2025-06-14 22:14:50", "msecs": 519.0, "correlation_id": "b8e8cf29", "levelname": "ERROR", "name": "storeapi.main", "lineno": 60, "message": "Status: 401 HTTP Exception: Could not validate credentials"}
{"asctime": "2025-06-14 22:14:50", "msecs": 521.0, "correlation_id": "b8e8cf29", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:10511 - \"POST /posts/ HTTP/1.1\" 401"}
{"asctime": "2025-06-14 22:17:57", "msecs": 2.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 263, "message": "Shutting down"}
{"asctime": "2025-06-14 22:17:57", "msecs": 105.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 67, "message": "Waiting for application shutdown."}
{"asctime": "2025-06-14 22:17:57", "msecs": 110.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 76, "message": "Application shutdown complete."}
{"asctime": "2025-06-14 22:17:57", "msecs": 112.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 93, "message": "Finished server process [24448]", "color_message": "Finished server process [\u001b[36m%d\u001b[0m]"}
{"asctime": "2025-06-14 22:17:59", "msecs": 728.0, "correlation_id": "-", "levelname": "INFO", "name": "storeapi.main", "lineno": 23, "message": "Starting application lifespan"}
{"asctime": "2025-06-14 22:17:59", "msecs": 732.0, "correlation_id": "-", "levelname": "DEBUG", "name": "storeapi.main", "lineno": 24, "message": "Connecting to the database"}
{"asctime": "2025-06-14 22:18:00", "msecs": 16.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 62, "message": "Application startup complete."}
{"asctime": "2025-06-14 22:18:00", "msecs": 17.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 215, "message": "Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)", "color_message": "Uvicorn running on \u001b[1m%s://%s:%d\u001b[0m (Press CTRL+C to quit)"}
{"asctime": "2025-06-14 22:18:03", "msecs": 975.0, "correlation_id": "12496128", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 113, "message": "Received token: eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MSwic3ViIjoicmFmaSIsImVtYWlsIjoicmFmaS5oYXNhbkBnbWFpbC5jb20iLCJleHAiOjE3NDk5MTczNzF9.tj-YVG02Tr4RkysDuyCHcleSgqPTXekltkLNMLCt690"}
{"asctime": "2025-06-14 22:18:03", "msecs": 982.0, "correlation_id": "12496128", "levelname": "ERROR", "name": "storeapi.security.security", "lineno": 125, "message": "JWT Error: Signature has expired."}
{"asctime": "2025-06-14 22:18:03", "msecs": 984.0, "correlation_id": "12496128", "levelname": "ERROR", "name": "storeapi.main", "lineno": 60, "message": "Status: 401 HTTP Exception: Could not validate credentials"}
{"asctime": "2025-06-14 22:18:03", "msecs": 985.0, "correlation_id": "12496128", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:10955 - \"POST /posts/ HTTP/1.1\" 401"}
{"asctime": "2025-06-14 22:19:00", "msecs": 793.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 263, "message": "Shutting down"}
{"asctime": "2025-06-14 22:19:00", "msecs": 897.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 67, "message": "Waiting for application shutdown."}
{"asctime": "2025-06-14 22:19:00", "msecs": 901.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 76, "message": "Application shutdown complete."}
{"asctime": "2025-06-14 22:19:00", "msecs": 902.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 93, "message": "Finished server process [24568]", "color_message": "Finished server process [\u001b[36m%d\u001b[0m]"}
{"asctime": "2025-06-14 22:19:03", "msecs": 360.0, "correlation_id": "-", "levelname": "INFO", "name": "storeapi.main", "lineno": 23, "message": "Starting application lifespan"}
{"asctime": "2025-06-14 22:19:03", "msecs": 364.0, "correlation_id": "-", "levelname": "DEBUG", "name": "storeapi.main", "lineno": 24, "message": "Connecting to the database"}
{"asctime": "2025-06-14 22:19:03", "msecs": 639.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 62, "message": "Application startup complete."}
{"asctime": "2025-06-14 22:19:03", "msecs": 641.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 215, "message": "Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)", "color_message": "Uvicorn running on \u001b[1m%s://%s:%d\u001b[0m (Press CTRL+C to quit)"}
{"asctime": "2025-06-14 22:19:09", "msecs": 478.0, "correlation_id": "b368ae62", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 74, "message": "Executing query to get user: SELECT users.id, users.username, users.email, users.hashed_password \nFROM users \nWHERE users.username = :username_1"}
{"asctime": "2025-06-14 22:19:09", "msecs": 483.0, "correlation_id": "b368ae62", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 78, "message": "Querying user with username: rafi"}
{"asctime": "2025-06-14 22:19:09", "msecs": 485.0, "correlation_id": "b368ae62", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 79, "message": "Query result: <databases.backends.common.records.Record object at 0x000001C18C20F9B0>"}
{"asctime": "2025-06-14 22:19:09", "msecs": 694.0, "correlation_id": "b368ae62", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:11008 - \"GET /users/login?username=rafi&password=12345%40 HTTP/1.1\" 200"}
{"asctime": "2025-06-14 22:19:29", "msecs": 719.0, "correlation_id": "855c586a", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 113, "message": "Received token: eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MSwic3ViIjoicmFmaSIsImVtYWlsIjoicmFmaS5oYXNhbkBnbWFpbC5jb20iLCJleHAiOjE3NDk5MTc5NjR9.rJMwnJcX7OX6EzGtEzA3r9-2nh2XPsB_ACnBpCVx5ao"}
{"asctime": "2025-06-14 22:19:29", "msecs": 721.0, "correlation_id": "855c586a", "levelname": "ERROR", "name": "storeapi.security.security", "lineno": 125, "message": "JWT Error: Signature has expired."}
{"asctime": "2025-06-14 22:19:29", "msecs": 722.0, "correlation_id": "855c586a", "levelname": "ERROR", "name": "storeapi.main", "lineno": 60, "message": "Status: 401 HTTP Exception: Could not validate credentials"}
{"asctime": "2025-06-14 22:19:29", "msecs": 724.0, "correlation_id": "855c586a", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:11021 - \"POST /posts/ HTTP/1.1\" 401"}
{"asctime": "2025-06-14 22:21:56", "msecs": 760.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 263, "message": "Shutting down"}
{"asctime": "2025-06-14 22:21:56", "msecs": 862.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 67, "message": "Waiting for application shutdown."}
{"asctime": "2025-06-14 22:21:56", "msecs": 867.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 76, "message": "Application shutdown complete."}
{"asctime": "2025-06-14 22:21:56", "msecs": 869.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 93, "message": "Finished server process [23828]", "color_message": "Finished server process [\u001b[36m%d\u001b[0m]"}
{"asctime": "2025-06-14 22:22:03", "msecs": 548.0, "correlation_id": "-", "levelname": "INFO", "name": "storeapi.main", "lineno": 23, "message": "Starting application lifespan"}
{"asctime": "2025-06-14 22:22:03", "msecs": 552.0, "correlation_id": "-", "levelname": "DEBUG", "name": "storeapi.main", "lineno": 24, "message": "Connecting to the database"}
{"asctime": "2025-06-14 22:22:03", "msecs": 837.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 62, "message": "Application startup complete."}
{"asctime": "2025-06-14 22:22:03", "msecs": 839.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 215, "message": "Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)", "color_message": "Uvicorn running on \u001b[1m%s://%s:%d\u001b[0m (Press CTRL+C to quit)"}
{"asctime": "2025-06-14 22:22:08", "msecs": 15.0, "correlation_id": "42ccdfe1", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 73, "message": "Executing query to get user: SELECT users.id, users.username, users.email, users.hashed_password \nFROM users \nWHERE users.username = :username_1"}
{"asctime": "2025-06-14 22:22:08", "msecs": 20.0, "correlation_id": "42ccdfe1", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 77, "message": "Querying user with username: rafi"}
{"asctime": "2025-06-14 22:22:08", "msecs": 22.0, "correlation_id": "42ccdfe1", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 78, "message": "Query result: <databases.backends.common.records.Record object at 0x000002341F4AF9B0>"}
{"asctime": "2025-06-14 22:22:08", "msecs": 235.0, "correlation_id": "42ccdfe1", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:11140 - \"GET /users/login?username=rafi&password=12345%40 HTTP/1.1\" 200"}
{"asctime": "2025-06-14 22:22:17", "msecs": 304.0, "correlation_id": "17c66cd6", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 112, "message": "Received token: eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MSwic3ViIjoicmFmaSIsImVtYWlsIjoicmFmaS5oYXNhbkBnbWFpbC5jb20iLCJleHAiOjE3NDk5MTkwMjh9.yPYpy4JtQqrfnQfJPIdJiE8zJOdYXQ_dGzxGvlVAk9E"}
{"asctime": "2025-06-14 22:22:17", "msecs": 307.0, "correlation_id": "17c66cd6", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 115, "message": "Decoded JWT payload: {'id': 1, 'sub': 'rafi', 'email': 'rafi.hasan@gmail.com', 'exp': 1749919028}"}
{"asctime": "2025-06-14 22:22:17", "msecs": 309.0, "correlation_id": "17c66cd6", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 73, "message": "Executing query to get user: SELECT users.id, users.username, users.email, users.hashed_password \nFROM users \nWHERE users.username = :username_1"}
{"asctime": "2025-06-14 22:22:17", "msecs": 312.0, "correlation_id": "17c66cd6", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 77, "message": "Querying user with username: rafi"}
{"asctime": "2025-06-14 22:22:17", "msecs": 313.0, "correlation_id": "17c66cd6", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 78, "message": "Query result: <databases.backends.common.records.Record object at 0x000002341F556690>"}
{"asctime": "2025-06-14 22:22:17", "msecs": 315.0, "correlation_id": "17c66cd6", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 119, "message": "Retrieved user: {'id': 1, 'username': 'rafi', 'email': 'rafi.hasan@gmail.com', 'hashed_password': '$2b$12$OWYaFk8zjX7IPOBHJLHfZuNfIs1m8FXQoT3GHAqGrc2sPtoEyjnEC'}"}
{"asctime": "2025-06-14 22:22:17", "msecs": 318.0, "correlation_id": "17c66cd6", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:11152 - \"POST /posts/ HTTP/1.1\" 500"}
{"asctime": "2025-06-14 22:22:17", "msecs": 319.0, "correlation_id": "17c66cd6", "levelname": "ERROR", "name": "uvicorn.error", "lineno": 414, "message": "Exception in ASGI application\n", "exc_info": "Traceback (most recent call last):\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\uvicorn\\protocols\\http\\httptools_impl.py\", line 409, in run_asgi\n    result = await app(  # type: ignore[func-returns-value]\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\uvicorn\\middleware\\proxy_headers.py\", line 60, in __call__\n    return await self.app(scope, receive, send)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\fastapi\\applications.py\", line 1054, in __call__\n    await super().__call__(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\applications.py\", line 112, in __call__\n    await self.middleware_stack(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\middleware\\errors.py\", line 187, in __call__\n    raise exc\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\middleware\\errors.py\", line 165, in __call__\n    await self.app(scope, receive, _send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\asgi_correlation_id\\middleware.py\", line 90, in __call__\n    await self.app(scope, receive, handle_outgoing_request)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\middleware\\cors.py\", line 85, in __call__\n    await self.app(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\middleware\\exceptions.py\", line 62, in __call__\n    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 53, in wrapped_app\n    raise exc\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 42, in wrapped_app\n    await app(scope, receive, sender)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 714, in __call__\n    await self.middleware_stack(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 734, in app\n    await route.handle(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 288, in handle\n    await self.app(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 76, in app\n    await wrap_app_handling_exceptions(app, request)(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 53, in wrapped_app\n    raise exc\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 42, in wrapped_app\n    await app(scope, receive, sender)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 73, in app\n    response = await f(request)\n               ^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\fastapi\\routing.py\", line 301, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\fastapi\\routing.py\", line 212, in run_endpoint_function\n    return await dependant.call(**values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\storeapi\\controller\\controller.py\", line 14, in create_post\n    new_post = {**post.model_dump(), \"user_id\": current_user.id}\n                                                ^^^^^^^^^^^^^^^\nAttributeError: 'dict' object has no attribute 'id'"}
{"asctime": "2025-06-14 22:24:55", "msecs": 895.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 263, "message": "Shutting down"}
{"asctime": "2025-06-14 22:24:55", "msecs": 999.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 67, "message": "Waiting for application shutdown."}
{"asctime": "2025-06-14 22:24:56", "msecs": 4.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 76, "message": "Application shutdown complete."}
{"asctime": "2025-06-14 22:24:56", "msecs": 6.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 93, "message": "Finished server process [26760]", "color_message": "Finished server process [\u001b[36m%d\u001b[0m]"}
{"asctime": "2025-06-14 22:24:58", "msecs": 644.0, "correlation_id": "-", "levelname": "INFO", "name": "storeapi.main", "lineno": 23, "message": "Starting application lifespan"}
{"asctime": "2025-06-14 22:24:58", "msecs": 648.0, "correlation_id": "-", "levelname": "DEBUG", "name": "storeapi.main", "lineno": 24, "message": "Connecting to the database"}
{"asctime": "2025-06-14 22:24:58", "msecs": 924.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 62, "message": "Application startup complete."}
{"asctime": "2025-06-14 22:24:58", "msecs": 926.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 215, "message": "Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)", "color_message": "Uvicorn running on \u001b[1m%s://%s:%d\u001b[0m (Press CTRL+C to quit)"}
{"asctime": "2025-06-14 22:25:00", "msecs": 407.0, "correlation_id": "e3e90007", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 112, "message": "Received token: eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MSwic3ViIjoicmFmaSIsImVtYWlsIjoicmFmaS5oYXNhbkBnbWFpbC5jb20iLCJleHAiOjE3NDk5MTkwMjh9.yPYpy4JtQqrfnQfJPIdJiE8zJOdYXQ_dGzxGvlVAk9E"}
{"asctime": "2025-06-14 22:25:00", "msecs": 413.0, "correlation_id": "e3e90007", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 115, "message": "Decoded JWT payload: {'id': 1, 'sub': 'rafi', 'email': 'rafi.hasan@gmail.com', 'exp': 1749919028}"}
{"asctime": "2025-06-14 22:25:00", "msecs": 415.0, "correlation_id": "e3e90007", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 73, "message": "Executing query to get user: SELECT users.id, users.username, users.email, users.hashed_password \nFROM users \nWHERE users.username = :username_1"}
{"asctime": "2025-06-14 22:25:00", "msecs": 421.0, "correlation_id": "e3e90007", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 77, "message": "Querying user with username: rafi"}
{"asctime": "2025-06-14 22:25:00", "msecs": 422.0, "correlation_id": "e3e90007", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 78, "message": "Query result: <databases.backends.common.records.Record object at 0x00000294441ED5B0>"}
{"asctime": "2025-06-14 22:25:00", "msecs": 424.0, "correlation_id": "e3e90007", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 119, "message": "Retrieved user: {'id': 1, 'username': 'rafi', 'email': 'rafi.hasan@gmail.com', 'hashed_password': '$2b$12$OWYaFk8zjX7IPOBHJLHfZuNfIs1m8FXQoT3GHAqGrc2sPtoEyjnEC'}"}
{"asctime": "2025-06-14 22:25:00", "msecs": 427.0, "correlation_id": "e3e90007", "levelname": "INFO", "name": "storeapi.controller.controller", "lineno": 18, "message": "Creating post for user: {'id': 1, 'username': 'rafi', 'email': 'rafi.hasan@gmail.com', 'hashed_password': '$2b$12$OWYaFk8zjX7IPOBHJLHfZuNfIs1m8FXQoT3GHAqGrc2sPtoEyjnEC'}"}
{"asctime": "2025-06-14 22:25:00", "msecs": 429.0, "correlation_id": "e3e90007", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:11314 - \"POST /posts/ HTTP/1.1\" 500"}
{"asctime": "2025-06-14 22:25:00", "msecs": 430.0, "correlation_id": "e3e90007", "levelname": "ERROR", "name": "uvicorn.error", "lineno": 414, "message": "Exception in ASGI application\n", "exc_info": "Traceback (most recent call last):\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\uvicorn\\protocols\\http\\httptools_impl.py\", line 409, in run_asgi\n    result = await app(  # type: ignore[func-returns-value]\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\uvicorn\\middleware\\proxy_headers.py\", line 60, in __call__\n    return await self.app(scope, receive, send)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\fastapi\\applications.py\", line 1054, in __call__\n    await super().__call__(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\applications.py\", line 112, in __call__\n    await self.middleware_stack(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\middleware\\errors.py\", line 187, in __call__\n    raise exc\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\middleware\\errors.py\", line 165, in __call__\n    await self.app(scope, receive, _send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\asgi_correlation_id\\middleware.py\", line 90, in __call__\n    await self.app(scope, receive, handle_outgoing_request)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\middleware\\cors.py\", line 85, in __call__\n    await self.app(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\middleware\\exceptions.py\", line 62, in __call__\n    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 53, in wrapped_app\n    raise exc\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 42, in wrapped_app\n    await app(scope, receive, sender)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 714, in __call__\n    await self.middleware_stack(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 734, in app\n    await route.handle(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 288, in handle\n    await self.app(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 76, in app\n    await wrap_app_handling_exceptions(app, request)(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 53, in wrapped_app\n    raise exc\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 42, in wrapped_app\n    await app(scope, receive, sender)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 73, in app\n    response = await f(request)\n               ^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\fastapi\\routing.py\", line 301, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\fastapi\\routing.py\", line 212, in run_endpoint_function\n    return await dependant.call(**values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\storeapi\\controller\\controller.py\", line 19, in create_post\n    new_post = {**post.model_dump(), \"user_id\": current_user.id}\n                                                ^^^^^^^^^^^^^^^\nAttributeError: 'dict' object has no attribute 'id'"}
{"asctime": "2025-06-14 22:25:49", "msecs": 655.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 263, "message": "Shutting down"}
{"asctime": "2025-06-14 22:25:49", "msecs": 758.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 67, "message": "Waiting for application shutdown."}
{"asctime": "2025-06-14 22:25:49", "msecs": 763.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 76, "message": "Application shutdown complete."}
{"asctime": "2025-06-14 22:25:49", "msecs": 764.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 93, "message": "Finished server process [28580]", "color_message": "Finished server process [\u001b[36m%d\u001b[0m]"}
{"asctime": "2025-06-14 22:25:52", "msecs": 422.0, "correlation_id": "-", "levelname": "INFO", "name": "storeapi.main", "lineno": 23, "message": "Starting application lifespan"}
{"asctime": "2025-06-14 22:25:52", "msecs": 425.0, "correlation_id": "-", "levelname": "DEBUG", "name": "storeapi.main", "lineno": 24, "message": "Connecting to the database"}
{"asctime": "2025-06-14 22:25:52", "msecs": 696.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 62, "message": "Application startup complete."}
{"asctime": "2025-06-14 22:25:52", "msecs": 698.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 215, "message": "Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)", "color_message": "Uvicorn running on \u001b[1m%s://%s:%d\u001b[0m (Press CTRL+C to quit)"}
{"asctime": "2025-06-14 22:25:55", "msecs": 77.0, "correlation_id": "919feba3", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 112, "message": "Received token: eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MSwic3ViIjoicmFmaSIsImVtYWlsIjoicmFmaS5oYXNhbkBnbWFpbC5jb20iLCJleHAiOjE3NDk5MTkwMjh9.yPYpy4JtQqrfnQfJPIdJiE8zJOdYXQ_dGzxGvlVAk9E"}
{"asctime": "2025-06-14 22:25:55", "msecs": 85.0, "correlation_id": "919feba3", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 115, "message": "Decoded JWT payload: {'id': 1, 'sub': 'rafi', 'email': 'rafi.hasan@gmail.com', 'exp': 1749919028}"}
{"asctime": "2025-06-14 22:25:55", "msecs": 87.0, "correlation_id": "919feba3", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 73, "message": "Executing query to get user: SELECT users.id, users.username, users.email, users.hashed_password \nFROM users \nWHERE users.username = :username_1"}
{"asctime": "2025-06-14 22:25:55", "msecs": 92.0, "correlation_id": "919feba3", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 77, "message": "Querying user with username: rafi"}
{"asctime": "2025-06-14 22:25:55", "msecs": 94.0, "correlation_id": "919feba3", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 78, "message": "Query result: <databases.backends.common.records.Record object at 0x000001C8DC1CD5B0>"}
{"asctime": "2025-06-14 22:25:55", "msecs": 95.0, "correlation_id": "919feba3", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 119, "message": "Retrieved user: {'id': 1, 'username': 'rafi', 'email': 'rafi.hasan@gmail.com', 'hashed_password': '$2b$12$OWYaFk8zjX7IPOBHJLHfZuNfIs1m8FXQoT3GHAqGrc2sPtoEyjnEC'}"}
{"asctime": "2025-06-14 22:25:55", "msecs": 97.0, "correlation_id": "919feba3", "levelname": "INFO", "name": "storeapi.controller.controller", "lineno": 18, "message": "Creating post for user: {'id': 1, 'username': 'rafi', 'email': 'rafi.hasan@gmail.com', 'hashed_password': '$2b$12$OWYaFk8zjX7IPOBHJLHfZuNfIs1m8FXQoT3GHAqGrc2sPtoEyjnEC'}"}
{"asctime": "2025-06-14 22:25:55", "msecs": 102.0, "correlation_id": "919feba3", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:11359 - \"POST /posts/ HTTP/1.1\" 201"}
{"asctime": "2025-06-14 22:28:55", "msecs": 556.0, "correlation_id": "87250a14", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 112, "message": "Received token: eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MSwic3ViIjoicmFmaSIsImVtYWlsIjoicmFmaS5oYXNhbkBnbWFpbC5jb20iLCJleHAiOjE3NDk5MTkwMjh9.yPYpy4JtQqrfnQfJPIdJiE8zJOdYXQ_dGzxGvlVAk9E"}
{"asctime": "2025-06-14 22:28:55", "msecs": 559.0, "correlation_id": "87250a14", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 115, "message": "Decoded JWT payload: {'id': 1, 'sub': 'rafi', 'email': 'rafi.hasan@gmail.com', 'exp': 1749919028}"}
{"asctime": "2025-06-14 22:28:55", "msecs": 561.0, "correlation_id": "87250a14", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 73, "message": "Executing query to get user: SELECT users.id, users.username, users.email, users.hashed_password \nFROM users \nWHERE users.username = :username_1"}
{"asctime": "2025-06-14 22:28:55", "msecs": 563.0, "correlation_id": "87250a14", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 77, "message": "Querying user with username: rafi"}
{"asctime": "2025-06-14 22:28:55", "msecs": 564.0, "correlation_id": "87250a14", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 78, "message": "Query result: <databases.backends.common.records.Record object at 0x000001C8DC1CE6F0>"}
{"asctime": "2025-06-14 22:28:55", "msecs": 566.0, "correlation_id": "87250a14", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 119, "message": "Retrieved user: {'id': 1, 'username': 'rafi', 'email': 'rafi.hasan@gmail.com', 'hashed_password': '$2b$12$OWYaFk8zjX7IPOBHJLHfZuNfIs1m8FXQoT3GHAqGrc2sPtoEyjnEC'}"}
{"asctime": "2025-06-14 22:28:55", "msecs": 568.0, "correlation_id": "87250a14", "levelname": "INFO", "name": "storeapi.controller.controller", "lineno": 18, "message": "Creating post for user: {'id': 1, 'username': 'rafi', 'email': 'rafi.hasan@gmail.com', 'hashed_password': '$2b$12$OWYaFk8zjX7IPOBHJLHfZuNfIs1m8FXQoT3GHAqGrc2sPtoEyjnEC'}"}
{"asctime": "2025-06-14 22:28:55", "msecs": 572.0, "correlation_id": "87250a14", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:11454 - \"POST /posts/ HTTP/1.1\" 201"}
{"asctime": "2025-06-14 22:29:02", "msecs": 542.0, "correlation_id": "781c6993", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:11463 - \"GET /posts/ HTTP/1.1\" 200"}
{"asctime": "2025-06-14 22:29:30", "msecs": 173.0, "correlation_id": "e604462b", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:11474 - \"POST /posts/1/comments HTTP/1.1\" 307"}
{"asctime": "2025-06-14 22:29:30", "msecs": 177.0, "correlation_id": "29c2334c", "levelname": "ERROR", "name": "storeapi.main", "lineno": 60, "message": "Status: 401 HTTP Exception: Not authenticated"}
{"asctime": "2025-06-14 22:29:30", "msecs": 178.0, "correlation_id": "29c2334c", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:11474 - \"POST /posts/1/comments/ HTTP/1.1\" 401"}
{"asctime": "2025-06-14 22:29:59", "msecs": 164.0, "correlation_id": "6f48c446", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 73, "message": "Executing query to get user: SELECT users.id, users.username, users.email, users.hashed_password \nFROM users \nWHERE users.username = :username_1"}
{"asctime": "2025-06-14 22:29:59", "msecs": 167.0, "correlation_id": "6f48c446", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 77, "message": "Querying user with username: sajid"}
{"asctime": "2025-06-14 22:29:59", "msecs": 168.0, "correlation_id": "6f48c446", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 78, "message": "Query result: None"}
{"asctime": "2025-06-14 22:29:59", "msecs": 169.0, "correlation_id": "6f48c446", "levelname": "INFO", "name": "storeapi.controller.user", "lineno": 32, "message": "Registering user: sajid"}
{"asctime": "2025-06-14 22:29:59", "msecs": 375.0, "correlation_id": "6f48c446", "levelname": "DEBUG", "name": "storeapi.controller.user", "lineno": 41, "message": "Executing query: INSERT INTO users (username, email, hashed_password) VALUES (:username, :email, :hashed_password)"}
{"asctime": "2025-06-14 22:29:59", "msecs": 377.0, "correlation_id": "6f48c446", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:11579 - \"POST /users/register HTTP/1.1\" 201"}
{"asctime": "2025-06-14 22:30:11", "msecs": 756.0, "correlation_id": "be4ef135", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 73, "message": "Executing query to get user: SELECT users.id, users.username, users.email, users.hashed_password \nFROM users \nWHERE users.username = :username_1"}
{"asctime": "2025-06-14 22:30:11", "msecs": 760.0, "correlation_id": "be4ef135", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 77, "message": "Querying user with username: sajid"}
{"asctime": "2025-06-14 22:30:11", "msecs": 761.0, "correlation_id": "be4ef135", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 78, "message": "Query result: <databases.backends.common.records.Record object at 0x000001C8DC230590>"}
{"asctime": "2025-06-14 22:30:11", "msecs": 948.0, "correlation_id": "be4ef135", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:11580 - \"GET /users/login?username=sajid&password=1234 HTTP/1.1\" 200"}
{"asctime": "2025-06-14 22:30:23", "msecs": 388.0, "correlation_id": "f7801adc", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:11637 - \"POST /posts/1/comments HTTP/1.1\" 307"}
{"asctime": "2025-06-14 22:30:23", "msecs": 392.0, "correlation_id": "32721675", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 112, "message": "Received token: eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6Miwic3ViIjoic2FqaWQiLCJlbWFpbCI6InNhamlkLmhhc2FuQGdtYWlsLmNvbSIsImV4cCI6MTc0OTkxOTUxMX0.KHrc0iSZuTEG0JP4WzENYzOm3lbAwhCRBhLDx70U_0w"}
{"asctime": "2025-06-14 22:30:23", "msecs": 395.0, "correlation_id": "32721675", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 115, "message": "Decoded JWT payload: {'id': 2, 'sub': 'sajid', 'email': 'sajid.hasan@gmail.com', 'exp': 1749919511}"}
{"asctime": "2025-06-14 22:30:23", "msecs": 397.0, "correlation_id": "32721675", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 73, "message": "Executing query to get user: SELECT users.id, users.username, users.email, users.hashed_password \nFROM users \nWHERE users.username = :username_1"}
{"asctime": "2025-06-14 22:30:23", "msecs": 400.0, "correlation_id": "32721675", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 77, "message": "Querying user with username: sajid"}
{"asctime": "2025-06-14 22:30:23", "msecs": 401.0, "correlation_id": "32721675", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 78, "message": "Query result: <databases.backends.common.records.Record object at 0x000001C8DC230D70>"}
{"asctime": "2025-06-14 22:30:23", "msecs": 403.0, "correlation_id": "32721675", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 119, "message": "Retrieved user: {'id': 2, 'username': 'sajid', 'email': 'sajid.hasan@gmail.com', 'hashed_password': '$2b$12$vtQ39ynNI3zJYsdgJ5yP3eRDntWT4yfSmnMp99.D4sWJ8Ag0OXvAS'}"}
{"asctime": "2025-06-14 22:30:23", "msecs": 404.0, "correlation_id": "32721675", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:11637 - \"POST /posts/1/comments/ HTTP/1.1\" 500"}
{"asctime": "2025-06-14 22:30:23", "msecs": 406.0, "correlation_id": "32721675", "levelname": "ERROR", "name": "uvicorn.error", "lineno": 414, "message": "Exception in ASGI application\n", "exc_info": "Traceback (most recent call last):\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\uvicorn\\protocols\\http\\httptools_impl.py\", line 409, in run_asgi\n    result = await app(  # type: ignore[func-returns-value]\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\uvicorn\\middleware\\proxy_headers.py\", line 60, in __call__\n    return await self.app(scope, receive, send)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\fastapi\\applications.py\", line 1054, in __call__\n    await super().__call__(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\applications.py\", line 112, in __call__\n    await self.middleware_stack(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\middleware\\errors.py\", line 187, in __call__\n    raise exc\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\middleware\\errors.py\", line 165, in __call__\n    await self.app(scope, receive, _send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\asgi_correlation_id\\middleware.py\", line 90, in __call__\n    await self.app(scope, receive, handle_outgoing_request)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\middleware\\cors.py\", line 85, in __call__\n    await self.app(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\middleware\\exceptions.py\", line 62, in __call__\n    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 53, in wrapped_app\n    raise exc\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 42, in wrapped_app\n    await app(scope, receive, sender)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 714, in __call__\n    await self.middleware_stack(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 734, in app\n    await route.handle(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 288, in handle\n    await self.app(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 76, in app\n    await wrap_app_handling_exceptions(app, request)(scope, receive, send)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 53, in wrapped_app\n    raise exc\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\_exception_handler.py\", line 42, in wrapped_app\n    await app(scope, receive, sender)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 73, in app\n    response = await f(request)\n               ^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\fastapi\\routing.py\", line 301, in app\n    raw_response = await run_endpoint_function(\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\fastapi\\routing.py\", line 212, in run_endpoint_function\n    return await dependant.call(**values)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\API\\FastAPI\\Project\\storeapi\\controller\\controller.py\", line 126, in add_comment\n    comment = {**comment.model_dump(), \"user_id\": current_user[\"id\"]}\n                                                  ^^^^^^^^^^^^^^^\nAttributeError: 'dict' object has no attribute 'id'"}
{"asctime": "2025-06-14 22:31:13", "msecs": 426.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 263, "message": "Shutting down"}
{"asctime": "2025-06-14 22:31:13", "msecs": 531.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 93, "message": "Finished server process [26884]", "color_message": "Finished server process [\u001b[36m%d\u001b[0m]"}
{"asctime": "2025-06-14 22:31:13", "msecs": 535.0, "correlation_id": "-", "levelname": "ERROR", "name": "uvicorn.error", "lineno": 134, "message": "Traceback (most recent call last):\n  File \"C:\\Users\\USER\\anaconda3\\Lib\\asyncio\\runners.py\", line 194, in run\n    return runner.run(main)\n           ^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\USER\\anaconda3\\Lib\\asyncio\\runners.py\", line 118, in run\n    return self._loop.run_until_complete(task)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\USER\\anaconda3\\Lib\\asyncio\\base_events.py\", line 674, in run_until_complete\n    self.run_forever()\n  File \"C:\\Users\\USER\\anaconda3\\Lib\\asyncio\\windows_events.py\", line 322, in run_forever\n    super().run_forever()\n  File \"C:\\Users\\USER\\anaconda3\\Lib\\asyncio\\base_events.py\", line 641, in run_forever\n    self._run_once()\n  File \"C:\\Users\\USER\\anaconda3\\Lib\\asyncio\\base_events.py\", line 1986, in _run_once\n    handle._run()\n  File \"C:\\Users\\USER\\anaconda3\\Lib\\asyncio\\events.py\", line 88, in _run\n    self._context.run(self._callback, *self._args)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\uvicorn\\server.py\", line 69, in serve\n    with self.capture_signals():\n         ^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\USER\\anaconda3\\Lib\\contextlib.py\", line 144, in __exit__\n    next(self.gen)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\uvicorn\\server.py\", line 330, in capture_signals\n    signal.raise_signal(captured_signal)\n  File \"C:\\Users\\USER\\anaconda3\\Lib\\asyncio\\runners.py\", line 157, in _on_sigint\n    raise KeyboardInterrupt()\nKeyboardInterrupt\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 699, in lifespan\n    await receive()\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\uvicorn\\lifespan\\on.py\", line 137, in receive\n    return await self.receive_queue.get()\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\USER\\anaconda3\\Lib\\asyncio\\queues.py\", line 158, in get\n    await getter\nasyncio.exceptions.CancelledError\n"}
{"asctime": "2025-06-14 22:31:15", "msecs": 651.0, "correlation_id": "-", "levelname": "INFO", "name": "storeapi.main", "lineno": 23, "message": "Starting application lifespan"}
{"asctime": "2025-06-14 22:31:15", "msecs": 656.0, "correlation_id": "-", "levelname": "DEBUG", "name": "storeapi.main", "lineno": 24, "message": "Connecting to the database"}
{"asctime": "2025-06-14 22:31:15", "msecs": 964.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 62, "message": "Application startup complete."}
{"asctime": "2025-06-14 22:31:15", "msecs": 966.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 215, "message": "Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)", "color_message": "Uvicorn running on \u001b[1m%s://%s:%d\u001b[0m (Press CTRL+C to quit)"}
{"asctime": "2025-06-14 22:31:22", "msecs": 138.0, "correlation_id": "05b6bedc", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:11849 - \"POST /posts/1/comments HTTP/1.1\" 307"}
{"asctime": "2025-06-14 22:31:22", "msecs": 145.0, "correlation_id": "d09409ff", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 112, "message": "Received token: eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6Miwic3ViIjoic2FqaWQiLCJlbWFpbCI6InNhamlkLmhhc2FuQGdtYWlsLmNvbSIsImV4cCI6MTc0OTkxOTUxMX0.KHrc0iSZuTEG0JP4WzENYzOm3lbAwhCRBhLDx70U_0w"}
{"asctime": "2025-06-14 22:31:22", "msecs": 154.0, "correlation_id": "d09409ff", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 115, "message": "Decoded JWT payload: {'id': 2, 'sub': 'sajid', 'email': 'sajid.hasan@gmail.com', 'exp': 1749919511}"}
{"asctime": "2025-06-14 22:31:22", "msecs": 156.0, "correlation_id": "d09409ff", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 73, "message": "Executing query to get user: SELECT users.id, users.username, users.email, users.hashed_password \nFROM users \nWHERE users.username = :username_1"}
{"asctime": "2025-06-14 22:31:22", "msecs": 160.0, "correlation_id": "d09409ff", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 77, "message": "Querying user with username: sajid"}
{"asctime": "2025-06-14 22:31:22", "msecs": 162.0, "correlation_id": "d09409ff", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 78, "message": "Query result: <databases.backends.common.records.Record object at 0x000001E996F999D0>"}
{"asctime": "2025-06-14 22:31:22", "msecs": 163.0, "correlation_id": "d09409ff", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 119, "message": "Retrieved user: {'id': 2, 'username': 'sajid', 'email': 'sajid.hasan@gmail.com', 'hashed_password': '$2b$12$vtQ39ynNI3zJYsdgJ5yP3eRDntWT4yfSmnMp99.D4sWJ8Ag0OXvAS'}"}
{"asctime": "2025-06-14 22:31:22", "msecs": 165.0, "correlation_id": "d09409ff", "levelname": "INFO", "name": "storeapi.controller.controller", "lineno": 126, "message": "Adding comment to post 1 by user 2"}
{"asctime": "2025-06-14 22:31:22", "msecs": 171.0, "correlation_id": "d09409ff", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:11849 - \"POST /posts/1/comments/ HTTP/1.1\" 201"}
{"asctime": "2025-06-14 22:31:45", "msecs": 878.0, "correlation_id": "36dd97fa", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:11869 - \"GET /posts/ HTTP/1.1\" 200"}
{"asctime": "2025-06-14 22:31:56", "msecs": 319.0, "correlation_id": "683c5a36", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:11890 - \"GET /posts/1 HTTP/1.1\" 200"}
{"asctime": "2025-06-14 22:32:27", "msecs": 73.0, "correlation_id": "c0f266b3", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:11926 - \"DELETE /posts/1/ HTTP/1.1\" 307"}
{"asctime": "2025-06-14 22:32:27", "msecs": 77.0, "correlation_id": "bd800e84", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 112, "message": "Received token: eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6Miwic3ViIjoic2FqaWQiLCJlbWFpbCI6InNhamlkLmhhc2FuQGdtYWlsLmNvbSIsImV4cCI6MTc0OTkxOTUxMX0.KHrc0iSZuTEG0JP4WzENYzOm3lbAwhCRBhLDx70U_0w"}
{"asctime": "2025-06-14 22:32:27", "msecs": 79.0, "correlation_id": "bd800e84", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 115, "message": "Decoded JWT payload: {'id': 2, 'sub': 'sajid', 'email': 'sajid.hasan@gmail.com', 'exp': 1749919511}"}
{"asctime": "2025-06-14 22:32:27", "msecs": 81.0, "correlation_id": "bd800e84", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 73, "message": "Executing query to get user: SELECT users.id, users.username, users.email, users.hashed_password \nFROM users \nWHERE users.username = :username_1"}
{"asctime": "2025-06-14 22:32:27", "msecs": 83.0, "correlation_id": "bd800e84", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 77, "message": "Querying user with username: sajid"}
{"asctime": "2025-06-14 22:32:27", "msecs": 85.0, "correlation_id": "bd800e84", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 78, "message": "Query result: <databases.backends.common.records.Record object at 0x000001E996F9AF30>"}
{"asctime": "2025-06-14 22:32:27", "msecs": 87.0, "correlation_id": "bd800e84", "levelname": "DEBUG", "name": "storeapi.security.security", "lineno": 119, "message": "Retrieved user: {'id': 2, 'username': 'sajid', 'email': 'sajid.hasan@gmail.com', 'hashed_password': '$2b$12$vtQ39ynNI3zJYsdgJ5yP3eRDntWT4yfSmnMp99.D4sWJ8Ag0OXvAS'}"}
{"asctime": "2025-06-14 22:32:27", "msecs": 90.0, "correlation_id": "bd800e84", "levelname": "ERROR", "name": "storeapi.main", "lineno": 60, "message": "Status: 403 HTTP Exception: You do not have permission to delete this post"}
{"asctime": "2025-06-14 22:32:27", "msecs": 91.0, "correlation_id": "bd800e84", "levelname": "INFO", "name": "uvicorn.access", "lineno": 476, "message": "127.0.0.1:11926 - \"DELETE /posts/1 HTTP/1.1\" 403"}
{"asctime": "2025-06-14 22:33:01", "msecs": 512.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 263, "message": "Shutting down"}
{"asctime": "2025-06-14 22:33:01", "msecs": 625.0, "correlation_id": "-", "levelname": "INFO", "name": "uvicorn.error", "lineno": 93, "message": "Finished server process [25812]", "color_message": "Finished server process [\u001b[36m%d\u001b[0m]"}
{"asctime": "2025-06-14 22:33:01", "msecs": 631.0, "correlation_id": "-", "levelname": "ERROR", "name": "uvicorn.error", "lineno": 134, "message": "Traceback (most recent call last):\n  File \"C:\\Users\\USER\\anaconda3\\Lib\\asyncio\\runners.py\", line 194, in run\n    return runner.run(main)\n           ^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\USER\\anaconda3\\Lib\\asyncio\\runners.py\", line 118, in run\n    return self._loop.run_until_complete(task)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\USER\\anaconda3\\Lib\\asyncio\\base_events.py\", line 674, in run_until_complete\n    self.run_forever()\n  File \"C:\\Users\\USER\\anaconda3\\Lib\\asyncio\\windows_events.py\", line 322, in run_forever\n    super().run_forever()\n  File \"C:\\Users\\USER\\anaconda3\\Lib\\asyncio\\base_events.py\", line 641, in run_forever\n    self._run_once()\n  File \"C:\\Users\\USER\\anaconda3\\Lib\\asyncio\\base_events.py\", line 1986, in _run_once\n    handle._run()\n  File \"C:\\Users\\USER\\anaconda3\\Lib\\asyncio\\events.py\", line 88, in _run\n    self._context.run(self._callback, *self._args)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\uvicorn\\server.py\", line 69, in serve\n    with self.capture_signals():\n         ^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\USER\\anaconda3\\Lib\\contextlib.py\", line 144, in __exit__\n    next(self.gen)\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\uvicorn\\server.py\", line 330, in capture_signals\n    signal.raise_signal(captured_signal)\n  File \"C:\\Users\\USER\\anaconda3\\Lib\\asyncio\\runners.py\", line 157, in _on_sigint\n    raise KeyboardInterrupt()\nKeyboardInterrupt\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\starlette\\routing.py\", line 699, in lifespan\n    await receive()\n  File \"C:\\API\\FastAPI\\Project\\.venv\\Lib\\site-packages\\uvicorn\\lifespan\\on.py\", line 137, in receive\n    return await self.receive_queue.get()\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\USER\\anaconda3\\Lib\\asyncio\\queues.py\", line 158, in get\n    await getter\nasyncio.exceptions.CancelledError\n"}
